blockbase/sass/base/_alignment.scss

```diff
@@ -1,18 +1,8 @@
-.wp-block-group.alignfull,
-*[class*="wp-container-"] //Anything that inherits layout (container)
-{
-	//give it some padding
-	padding-left: var(--wp--custom--post-content--padding--left);
-	padding-right: var(--wp--custom--post-content--padding--right);
-
-	//Any nested containers, and anything that is alignfull
-	*[class*="wp-container-"], // Any nested containers
-	> .alignfull { // Any direct descendant that is alignfull
-		// bust out of the container's padding
-		margin-left: calc(-1 * var(--wp--custom--post-content--padding--left)) !important;
-		margin-right: calc(-1 * var(--wp--custom--post-content--padding--right)) !important;
-		width: calc( 100% + var(--wp--custom--post-content--padding--left) + var(--wp--custom--post-content--padding--right) ) !important;
-	}
+p {
+	// An alternative to padding.
+	margin-left: auto;
+	margin-right: auto;
+	width: calc( 100vw - var(--wp--custom--post-content--padding--left) - var(--wp--custom--post-content--padding--right) );
```

Comment:
Setting a width here causes paragraphs to sometime overflow out of their containers:

<img width="945" alt="Screen Shot 2021-08-23 at 2 28 56 PM" src="https://user-images.githubusercontent.com/5375500/130498442-84fb63cb-8c14-4acb-9e28-8b4dc7f9df4e.png">

Reply:
yeah that makes sense. i think we should just remove it all